src/web/command_control/client/components/SettingsPanel.tsx

```diff
@@ -64,6 +67,18 @@ export function SettingsPanel(props: Props) {
         }
     }
     
+    
+    /**
+     * Event handler for the CSV dowload button.  Creates the CSV file and initiates
+     * the download.
+     *
+     * @async
```

Comment:
We can remove it.  It could be useful, but it's not in our current specs for function comments.  Too bad the IDE doesn't show async on hover, because that would be useful in practice.